python_modules/dagster/dagster/_core/definitions/asset_reconciliation_sensor.py

```diff
@@ -44,6 +45,73 @@
     from dagster._utils.caching_instance_queryer import CachingInstanceQueryer  # expensive import
 
 
+def get_implicit_auto_materialization_policy(
+    asset_graph: AssetGraph, asset_key: AssetKey
+) -> Optional[AutoMaterializePolicy]:
+    """For backcompat with pre-auto materialization policy graphs, assume a default scope of 1 day.
+    """
+    auto_materialization_policy = asset_graph.get_auto_materialize_policy(asset_key)
+    if auto_materialization_policy is None:
+        return AutoMaterializePolicy(
+            on_missing=True,
+            on_upstream_data_newer=not bool(
+                asset_graph.get_downstream_freshness_policies(asset_key=asset_key)
+            ),
+            for_freshness=True,
+            time_window_partition_scope_minutes=24 * 60,
+        )
+    return auto_materialization_policy
+
+
+def reconciliation_window_for_time_window_partitions(
+    partitions_def: TimeWindowPartitionsDefinition,
+    time_window_partition_scope: Optional[datetime.timedelta],
+    evaluation_time: datetime.datetime,
```

Comment:
Even though we probably should have used `evaluation_time` in the first place, `current_time` would be more consistent and thus make it easier to understand what this refers to.